src/components/CameraPermissions/Primer/index.tsx

```diff
@@ -11,34 +11,58 @@ import type { WithLocalisedProps, WithTrackingProps } from '~types/hocs'
 
 type PermissionsProps = {
   onNext: () => void
+  audio?: boolean
 }
 
 type Props = PermissionsProps & WithLocalisedProps & WithTrackingProps
 
-const Permissions: FunctionComponent<Props> = ({ onNext, translate }) => (
-  <div className={theme.fullHeightContainer}>
-    <PageTitle
-      title={translate('permission.title_cam')}
-      subTitle={translate('permission.subtitle_cam')}
-    />
-    <div className={classNames(style.bodyWrapper, theme.scrollableContent)}>
-      <div className={style.image}>
-        <div className={style.graphic} />
+const localeKeys = {
+  camera: {
+    title: 'permission.title_cam',
+    subTitle: 'permission.subtitle_cam',
+    body: 'permission.body_cam',
+    button: 'permission.button_primary_cam',
+  },
+  microphoneAndCamera: {
+    title: 'permission.title_microphone_cam',
+    subTitle: 'permission.subtitle_microphone_cam',
+    body: 'permission.body_microphone_cam',
+    button: 'permission.button_primary_microphone_cam',
```

Comment:
Did any locale key or value change? If we add/remove/change keys or value we need to update the migration file